packages/headless/src/utils/utils.ts

```diff
@@ -17,16 +17,22 @@ export function isEmptyString(str: string) {
 
 export function removeDuplicates<T>(
```

Comment:
I don't think we already have a UT for this file, but we should add one.

And we should test the function for both left/right